src/templates/index.jade

```diff
@@ -7,19 +7,65 @@ html
         include socialmetatags
         link(rel='stylesheet' type='text/css' href='public/style.css')
         script(src='public/head.js')
-
+        script(src='https://maps.googleapis.com/maps/api/js')
```

Comment:
Don't put JS's here. They are heavy and they prevent DOM to render quicky.

Almost all JS should be put in one file which is loaded as last element on the site, after DOM render.

Check how it is done with `concat` and `uglify` task in `Gruntfile.js`.

And one more thing! Don't load Google Maps at start. Load it only when user wanted that. Otherwise we will make users load mmuuuuuuuch unless data on mobile which is not so good.